core/src/execution/proto/expr.proto

```diff
@@ -219,6 +219,7 @@ message Subtract {
   Expr right = 2;
   bool fail_on_error = 3;
   DataType return_type = 4;
+  EvalMode eval_mode = 5;
```

Comment:
Just curious, if a Substrait plan was to send one of the options how it would map to this new functionality?  How does ANSI map to an option?

https://substrait.io/extensions/functions_arithmetic/#subtract